### PR TITLE
CS falcon list files add limitation docs

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-38868

## Description
Added a comment about known limitation in `!cs-falcon-list-files` command.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml [hunk 1]
```diff
@@ -600,7 +600,7 @@ script:
       description: The size of the file in bytes.
       type: Number
   - arguments: []
-    description: Returns a list of put-file IDs that are available for the user in the 'put' command.
+    description: Returns a list of put-file IDs that are available for the user in the 'put' command. Due to an API limitation the maximum number of files returned is 100.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml:603]:
```suggestion
    description: Returns a list of put-file IDs that are available for the user in the 'put' command. Due to an API limitation, the maximum number of files returned is 100.
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 2]
```diff
@@ -763,7 +763,7 @@ Returns files based on the provided IDs. These files are used for the RTR 'put'
 ### 12. cs-falcon-list-files
 
 ---
-Returns a list of put-file ID's that are available for the user in the `put` command.
+Returns a list of put-file ID's that are available for the user in the `put` command. Due to an API limitation the maximum number of files returned is 100.
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:766]:
```suggestion
Returns a list of put-file ID's that are available for the user in the `put` command. Due to an API limitation, the maximum number of files returned is 100.
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by thefrieddan1 — APPROVED** [review]

**Comment by MosheEichler** [comment]
@DeanArbel Needs a force merge [fails](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/6761193/raw#L1127) on `Host ID <sha> - Could not establish sensor comms`

**Pull request merged**